### ISSUE TITLE
Weird merging of downloaded chapters.

### ISSUE THREAD

**Issue by typemobszn** [Aidoku/Aidoku#399]
I had two separate titles (completely unrelated to each other) downloaded from the same source (mangakatana) and upon starting to read ch 1 of one of the titles the titles merged(?). I'm attaching pictures to make it easier to understand.

I have been unable to replicate this rare issue, maybe I had downloaded them both at the same time?

Deleting the downloading chapters of one of the titles fixed the issue.

![Screenshot 2024-02-09 145049](https://github.com/Aidoku/Aidoku/assets/60714832/d1556a0f-c118-435c-bb35-6525f8e90431)

**Comment by FoxyDr3am** [comment]
Oh it happen to me a couple of times last year 
On both my phone 13PM and 15PM
But lately it's doesn't happen anymore

**Comment by typemobszn** [comment]
> Oh it happen to me a couple of times last year On both my phone 13PM and 15PM But lately it's doesn't happen anymore

Did you have the chapters downloaded as well? If yes same source?

**Comment by FoxyDr3am** [comment]
Yeah I downloaded the chapters but I wasn't using the same source and I think I used 2 different sources and they got mixed for no reason
Since it hasn't happened to me for an long I thought they fix it

**Comment by Skittyblock** [comment]
hoping this was fixed with #779, but i haven't seen this issue ever happen myself. feel free to reopen if you see it again or have steps to reproduce
